### ISSUE TITLE
Bad output on the disk safety check in Alba

### ISSUE THREAD

**Issue by matthiasdeblock** [openvstorage/openvstorage-health-check#282]
Bad output on the disk safety check in Alba:

```
[INFO] Storagerouter Type: MASTER
[INFO] Environment Os: Ubuntu 16.04 xenial
[INFO] Environment Release: Fargo
[INFO] Hostname: stor-01.be-g8-4
[INFO] Cluster Id: 0DV7F8opVwsH0JGo
[INFO] Storagerouter Id: jJ4T80wcKdJ2rtxg
[INFO] Starting OpenvStorage Healthcheck version 3.2.0-fargo.2-1
[INFO] ======================

[WARNING] The disk safety of 94 namespace(s) is 3, max. disk safety is 4:
d5530e78-c773-4225-b3c2-1d9b7f8348e5 with 100% of its objects,
db889952-69e6-48ee-917d-65cac4e57dba with 100% of its objects,
df22a5b4-1ed5-4363-88f9-bc8d8b7ee7fa with 252100% of its objects,
df22a5b4-1ed5-4363-88f9-bc8d8b7ee7fa with 0% of its objects,
e695f8de-0447-46cb-979f-ddd118ad4700 with 100% of its objects,
e706ce67-f43f-4e71-b60d-6b3addd4e858 with 100% of its objects,
e7477b05-8029-49f3-9988-55d514ef5c54 with 2766400% of its objects,
ebeb8976-e97d-46cc-aae1-a4567ce2c259 with 1695000% of its objects,
ec7930f9-b8a6-4037-9bfc-6f695215df23 with 100% of its objects,
```

**Comment by kinvaris** [comment]
This is a calculation and interpretation error. Should be fixed in the following pull request: https://github.com/openvstorage/openvstorage-health-check/pull/284

**Comment by kinvaris** [comment]
fixed in `3.2.0-fargo.3-1`
